libs/workordersidesheet/src/lib/utils-sidesheet/useGetEchoConfig.ts

```diff
@@ -0,0 +1,41 @@
+import { useContextId } from '@cc-components/shared';
+import { useHttpClient } from '@equinor/fusion-framework-react-app/http';
+import { useQuery } from '@tanstack/react-query';
+
+export const useGetEchoConfig = (packageId: string) => {
+  const client = useHttpClient('cc-app');
```

Comment:
Prefer the useHttpClient from the @cc-components/shared scope. Using that you will avoid the string values in the parameter. "cc-app"

Reply:
Fixed